Makefile

```diff
@@ -571,7 +574,18 @@ generate-metrics-config: $(ENVSUBST_BIN) ## Generate ./hack/observability/kube-s
 
 .PHONY: generate-diagrams
 generate-diagrams: ## Generate diagrams for *.plantuml files
-	$(MAKE) -C docs diagrams
+	$(MAKE) generate-diagrams-book
+	$(MAKE) generate-diagrams-proposals
+
+.PHONY: generate-diagrams-book
+generate-diagrams-book: ## Generate diagrams for *.plantuml files in book
```

Comment:
Do you know why we run plantuml on markdown files?

Reply:
Appears to have been introduced here: https://github.com/kubernetes-sigs/cluster-api/pull/7033 but it looks like there no actual images generated from it so I've dropped them now.